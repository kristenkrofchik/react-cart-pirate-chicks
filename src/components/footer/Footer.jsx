import React from 'react';
import './Footer.styles.scss';

const Footer = () => {
    const year = new Date().getFullYear();
    return (
        <div className='footer'>
            {year} Â© Pirate Chicks Vintage
        </div>
    );
}

export default Footer;