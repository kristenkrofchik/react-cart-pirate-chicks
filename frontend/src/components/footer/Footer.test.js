import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

it("matches snapshot", function () {
    const { asFragment } = render(
        <Footer />
    );
    expect(asFragment()).toMatchSnapshot();
  });