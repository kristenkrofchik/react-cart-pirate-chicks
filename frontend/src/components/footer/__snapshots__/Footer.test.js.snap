// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    2021 Â© Pirate Chicks Vintage
  </div>
</DocumentFragment>
`;
